src/libworld_test/libworld_test.h

```diff
@@ -449,7 +449,7 @@ namespace world
                 airports.push_back(factory(testHost));
             }
 
-            auto world = WorldBuilder::assembleSampleWorld(testHost, airports);
+            auto world = WorldBuilder::assembleSampleWorld(testHost, airports, shared_ptr<RandomRouteProvider>(NULL));
```

Comment:
Just passing `nullptr` instead of `shared_ptr<RandomRouteProvider>(NULL)` should work. 
BTW, AFAIK, in general, `nullptr` is preferred over `NULL`

Reply:
I'm trying my best to do the things properly but my mainly C experience is sometimes taking over :smiley:.